packages/client-sdk-nodejs/src/config/configuration.ts

```diff
@@ -77,6 +84,33 @@ export interface Configuration {
    */
   addMiddleware(middleware: Middleware): Configuration;
 
+  /**
```

Comment:
let's not expose these today. Increases API surface area and we don't have a concrete customer use case yet. We can easily add them later if something justifies it.